File: 10818 최소, 최대/10818.py
n = int(input())
array = map(int, input().split(' '))

maxValue = -987654231
minValue = 987654321
for num in array:
    if maxValue < num:
        maxValue = num
    if minValue > num:
        minValue = num

print('{0} {1}'.format(minValue, maxValue))

File: 10818 최소, 최대/10818.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');

let n = parseInt(input[0]);
let array = input[1].split(' ');
let max = -987654321;
let min = 987654321;

for (let i = 0; i < n; ++i) {
    let number = parseInt(array[i]);
    if (number < min) {
        min = number;
    }
    if (number > max) {
        max = number;
    }
}

console.log(min + ' ' + max);